src/libraries/System.Security.Cryptography.X509Certificates/tests/CollectionTests.cs

```diff
@@ -1640,6 +1690,71 @@ private static void TestExportStore(X509ContentType ct)
             }
         }
 
+        private static void AssertPemExport(X509Certificate2Collection collection, string expectedContents)
+        {
+            Span<char> buffer = new char[expectedContents.Length];
```

Comment:
Do we think "bigger than needed" is an interesting case here?

Reply:
Eh, interesting enough to make you leave a remark, so, let's say yes and add a test case for it.